### ISSUE TITLE
User specific canned replies

### ISSUE THREAD

**Issue by cklosowski** [pippinsplugins/bbPress-Canned-Replies#1]
I think it would be nice if a user could set the canned reply to be specific to them only. Possibly using the 'private' attribute, or even a post meta field to allow that canned reply to only be edited/used by the user who created it?

It should also allow the setting to be removed, which would then make that reply available to all users who can use canned replies.

Admins would have full access to these private canned replies in the back end (for editing and removal in the case that the original author is no longer a user), but not have them listed on the front end.

What do you think? If this is something you think belongs in here, I'll try and code something up.

**Comment by pippinsplugins** [comment]
Hmm, interesting. Are you thinking primarily for multi-admin/moderator forums?

**Comment by cklosowski** [comment]
Yep. That's the thought. When I did email support we used something like this (a Firefox plugin) that way we could use our own signature and a standard template without all having the same opening, closing, and could vary the wording slightly for responses.
